diagnostic.md

```diff
@@ -27,7 +33,15 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$base-font-size: 30%;
+
+element {
+  font-size: $base-font-size
+}
+
+h1 {
+  font-size: $base-font-size
```

Comment:
This should be `font-size: $base-font-size * 1.3;` since we're increasing it by 30%.